middleware/logging.go

```diff
@@ -34,6 +34,11 @@ func (rw *responseWriter) WriteHeader(code int) {
 	rw.wroteHeader = true
 }
 
+func (rw *responseWriter) Flush() {
```

Comment:
Is this used implicitly somewhere?

Reply:
Yeah https://github.com/livepeer/catalyst-api/blob/6b158fe32d216fb5278756f770bceaac1d3f85d0/mapic/mistapiconnector_app.go#L368-L371